src/__tests__/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "moduleResolution": "nodenext",
```

Comment:
Curious, what does this do?

Reply:
It instructs TypeScript to use Node.js-style module resolution for the tests.